docs/Documentation/Updating/Migration.md

```diff
@@ -44,6 +49,33 @@ Changes introduced in:
   with any file and folder names you want. Only the `package.yml` is reserved as indicator for a quest
   package. [DOCS](../Reference.md#packages)
 
-### 2.0.0-DEV-98
-All existing RPGMenu users must update their RPGMenu config file. Simply rename it from `rpgmenu.config.yml` to
-`menuConfig.yml`.
+### _latest_
+- All your static events need to be converted to the new schedules system.
+  The [`realtime-daily`](../Schedules.md#daily-realtime-schedule-realtime-daily) schedule makes this easy:
+  ```YAML title="Old Syntax"
+  static:
+    '09:00': beton
+    '11:23': some_command,command_announcement
+  ```
+  ```YAML title="New Syntax"
+  schedules:
+    betonAt09: #(1)!
+      type: realtime-daily #(2)!
+      time: '09:00' #(3)!
+      events: beton #(4)!
+    cmdAt1123:
+      type: simple
+      time: '11:23'
+      events: some_command,command_announcement
+  ```
+  1. A name for the new schedule.  
+     Can be anything you want for organizing your schedules.
+
+  2. The type schedule `realtime-daily` was created for easy updating.   
+     It behaves just like the old static events.
+
+  3. The former key is now the time value.  
+     You still have to put it in 'quotes'.
+
+  4. The former value is now the events value.
```

Comment:
This hints are not correctly formatted and so they are below the code block and not as a hint in the code block